book/src/volume-cloning.md

```diff
@@ -0,0 +1,56 @@
+# Volume Cloning
+
+## Status and Releases
+
+** Status:** Alpha
+
+Compatible with CSI Version | Supported CSI external Provisioner Version | Min k8s Version | Max k8s version
+--|--|--|--|--|--
+v1.0  | v1.3 | v1.15 | -
```

Comment:
We've been leaving out the 'v' in the other pages

Reply:
Ok, the only page I saw using this at all (first glance any way) was the external-resizer page https://kubernetes-csi.github.io/docs/external-resizer.html

I'll remove them; this pr has me thinking we could use some doc writing guides and clean up quite a few inconsistencies.

Reply:
bleh. All the "feature" pages leave out the 'v': https://kubernetes-csi.github.io/docs/topology.html

We'll have to do another sweep later and make them consistent.